cookbook/core/control_flow/subworkflows.py

```diff
@@ -100,6 +110,86 @@ def nested_parent_wf(a: int) -> Tuple[int, str, str, str]:
 
 
 # %%
+# You can execute the nested workflows locally as well.
 # You can execute the nested workflows locally as well
 if __name__ == "__main__":
     print(f"Running nested_parent_wf(a=3) {nested_parent_wf(a=3)}")
+
+
+# %%
+# Child Workflow
```

Comment:
Child workflow is now "external workflow". See
https://unionai.slack.com/archives/C01HA7KQP7X/p1643732197015639?thread_ts=1643220389.065600&cid=C01HA7KQP7X.